test.rb

```diff
@@ -0,0 +1 @@
+require 'test'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.